### PR TITLE
ci: cache efa installer

### PR DESCRIPTION
**Pull request by aws-nslick**
*Description of changes:*

Once a day, download efa installer and store it in actions cache. Other actions can use this instead of downloading from network.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by rajachan — DISMISSED** [review]

**Review by rajachan — APPROVED** [review]

**Comment by aws-nslick** [comment]
Not waiting for CI as this is completely independent of everything else, to unblock other PRs.

**Pull request merged**
